dev/ledger.md

```diff
@@ -411,6 +412,17 @@ transaction.  This is determined by the _signature_ of a transaction:
 
    - The multisignature version _v_ (current value is 1).
 
+- A valid logic-signed transaction's signature is the _lsig_ object containing
+  the following fields:
+
+  - The logic _l_ which is versioned bytecode. (See TEAL docs for details)
+
+  - An optional single signature _sig_ of 64 bytes valid for the sender of the transaction which has signed the bytes in _l_.
```

Comment:
Indicate that the signature only covers the code _l_ and not the arguments

Reply:
this already says precisely that

Reply:
After "- An optional array of byte strings _arg_ which are arguments supplied to the program in _l_." add "This value (if present) is not covered under the signature or multisignature field above."

Reply:
noted that arg bytes are not covered by sigs